pkg/replicatedapp/api.go

```diff
@@ -165,3 +171,46 @@ func getApplicationMetadataFromHost(host string, endpoint string, upstream *url.
 
 	return respBody, nil
 }
+
+func SendApplicationMetricsData(license *kotsv1beta1.License, app *apptypes.App, data map[string]interface{}) error {
+	url := fmt.Sprintf("%s/application/metrics", license.Spec.Endpoint)
```

Comment:
i know this isn't defined in KOTS, but i wonder if it should be `/application/custom-metrics`